src/IonParserBinaryRaw.ts

```diff
@@ -131,6 +115,17 @@ const ivm_image_1 = IVM.binary[1];
 const ivm_image_2 = IVM.binary[2];
 const ivm_image_3 = IVM.binary[3];
 
+// stack pointer for _ts to store length and type information of container
+class stackPointer {
```

Comment:
To me, the phrase "stack pointer" means the address of the top of a stack. Since we'll have an instance of this type for each container we step into, I suggest `EncodingContainer`, `EncodingLevel` or similar. Note that in TypeScript's style conventions, [class names always begin with a capital letter](https://basarat.gitbook.io/typescript/styleguide#class).